selenium_webdriver/src/main/java/core/driver/DriverSingleton.java

```diff
@@ -0,0 +1,33 @@
+package core.driver;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+public class DriverSingleton {
```

Comment:
I'm not sure it's a Singleton pattern. Please look carefully at what variables should be private and static. Also you have to have a private Contructor

Reply:
Fixed